LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2018, HTSJDK-Next Authors
```

Comment:
Should we open an issue to rename `HTSJDK-Next` to the final project name? Would it be `htsjdk`3` or a completely new name (such as the new Hadoop-BAM project)?

Reply:
That's a good question.  I think keeping the htsjdk name is useful since it's well known and this is going to be closely related to htsjdk.

Reply:
I suggest to use `htsjdk3` for the repo name and maintain the namespace

Reply:
I want to change to use a proper package name.  The current namespace is problematic since certain java tools assume there's a root level package below all your code.  I was hoping to go with org.samtools.htsjdk, but we don't own that url and there are currently domain squatters on it.  So I'm not sure what to do.  Mabe org.htsjdk